### PR TITLE
[Material] Added support to has no control (continue/cancel button) in Stepper Widget

### PR DESCRIPTION
**Pull request by ViniciusSossela**
Flutter stepper widget does not allow to has no control button (continue/cancel).

Solution: I have created a `bool useControl` property to allow developer choose between use or not control's button.

Case of use: Product delivery stepper is not controlled by users, the app/system controls it's delivery.

### CODE REVIEW

**Comment by ViniciusSossela** [comment]
> Thanks for your pull request. It looks like this may be your first contribution to a Google open source project (if not, look below for help). Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).
> 
> 📝 **Please visit https://cla.developers.google.com/ to sign.**
> 
> Once you've signed (or fixed any issues), please reply here (e.g. `I signed it!`) and we'll verify it.
> 
> #### What to do if you already signed the CLA
> ##### Individual signers
> * It's possible we don't have your GitHub username or you're using a different email address on your commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> 
> ##### Corporate signers
> * Your company has a Point of Contact who decides which employees are authorized to participate. Ask your POC to be added to the group of authorized contributors. If you don't know who your Point of Contact is, direct the Google project maintainer to [go/cla#troubleshoot](http://go/cla#troubleshoot) ([Public version](https://opensource.google.com/docs/cla/#troubleshoot)).
> * The email used to register you as an authorized contributor must be the email used for the Git commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> * The email used to register you as an authorized contributor must also be [attached to your GitHub account](https://github.com/settings/emails).

**Comment by ViniciusSossela** [comment]
> Thanks for your pull request. It looks like this may be your first contribution to a Google open source project (if not, look below for help). Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).
> 
> 📝 **Please visit https://cla.developers.google.com/ to sign.**
> 
> Once you've signed (or fixed any issues), please reply here (e.g. `I signed it!`) and we'll verify it.
> 
> #### What to do if you already signed the CLA
> ##### Individual signers
> * It's possible we don't have your GitHub username or you're using a different email address on your commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> 
> ##### Corporate signers
> * Your company has a Point of Contact who decides which employees are authorized to participate. Ask your POC to be added to the group of authorized contributors. If you don't know who your Point of Contact is, direct the Google project maintainer to [go/cla#troubleshoot](http://go/cla#troubleshoot) ([Public version](https://opensource.google.com/docs/cla/#troubleshoot)).
> * The email used to register you as an authorized contributor must be the email used for the Git commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> * The email used to register you as an authorized contributor must also be [attached to your GitHub account](https://github.com/settings/emails).

I signed it!

**Comment by ViniciusSossela** [comment]
@HansMuller and @dnfield any news on this?

**Comment by Hixie** [comment]
It seems like this is already possible by just overriding `controlsBuilder` to return a `SizedBox.shrink` or some such. Maybe we should just support `controlsBuilder` returning null, and have it skip the controls if it returns null?

File: packages/flutter/lib/src/material/stepper.dart [hunk 1]
```diff
@@ -563,6 +566,12 @@ class _StepperState extends State<Stepper> with TickerProviderStateMixin {
     );
   }
 
+  List<Widget> _buildVerticalControlsBody(int index) {
+    return widget.useControl
+        ? <Widget>[widget.steps[index].content, _buildVerticalControls()]
+        : <Widget>[Row(children: <Widget>[Container(child: widget.steps[index].content)])];
```
Comment (Hixie) [inline packages/flutter/lib/src/material/stepper.dart]: the row and the container seem like they should be doing nothing here, what is their purpose?

Comment (ViniciusSossela) [reply packages/flutter/lib/src/material/stepper.dart]: The row and container was added to keep the Stepper layout looking fine, without it the Stepper breaks it's alignment as showed in the bellow image:
The same issue happens if we make 'controlsBuilder' return `SizedBox.shrink`

screen shot 2018-12-01 at 14 47 30



screenshot_1543682947

**Comment by ViniciusSossela** [comment]
> It seems like this is already possible by just overriding `controlsBuilder` to return a `SizedBox.shrink` or some such. Maybe we should just support `controlsBuilder` returning null, and have it skip the controls if it returns null?

Hey @Hixie I just implemented in this way, now `controlsBuilder` supports returning null, and if it returns null we do not build the control... What do you think? Also I needed to put `widget.steps[index].content` in a Row otherwise it breaks the layout, in case we have `controlsBuilder != null` than the Row isn't needed because control already has it's own Row inside a Container

Here are one screenshoot with the result:
screenshot_1543687112

**Pull request closed without merging**